packages/appstore/src/components/modals/mt5-migration-modal/mt5-migration-modal.tsx

```diff
@@ -0,0 +1,279 @@
+            <React.Fragment>
+                <div>
+                    <div className='mt5-migration-modal__description'>
+                        <Text as='p' color='general' size={header_size} align='center' weight='bold'>
+                            <Localize i18n_default_text='What will happen to the funds in my existing account(s)?' />
+                        </Text>
+                    </div>
+                    <div className='mt5-migration-modal__existing-accounts'>
+                        <div className='mt5-migration-modal__existing-accounts-card'>
+                            <div className='mt5-migration-modal__existing-accounts-card-content'>
+                                <Text as='div' weight='bold' size={title_size}>
+                                    <Localize i18n_default_text='If you have open positions' />
+                                </Text>
+
+                                <MigrationInfoMessages text_message='Your funds will remain in your existing MT5 account(s).' />
+
+                                <MigrationInfoMessages text_message='You can continue trading on your existing MT5 account(s) until you close all open positions.' />
+
+                                <MigrationInfoMessages text_message='New MT5 account(s) under the BVI jurisdiction will be created for future trades.' />
+                            </div>
+                        </div>
+                        <div className='mt5-migration-modal__existing-accounts-card'>
+                            <div className='mt5-migration-modal__existing-accounts-card-content'>
+                                <Text weight='bold' size={title_size}>
+                                    <Localize i18n_default_text='If you have open positions' />
+                                </Text>
+
+                                <MigrationInfoMessages text_message='Your funds will remain in your existing MT5 account(s).' />
+
+                                <MigrationInfoMessages text_message='You can continue trading on your existing MT5 account(s) until you close all open positions.' />
+
+                                <MigrationInfoMessages text_message='New MT5 account(s) under the BVI jurisdiction will be created for future trades.' />
+                            </div>
+                        </div>
+                    </div>
+                    <div>
+                        <div className=' mt5-migration-modal__existing-accounts-card-content'>
+                            <Checkbox
+                                value={is_checked}
+                                onChange={() => setIsChecked(true)}
+                                label={
+                                    <Text as='p' size={checkbox_text_size} line_height='xs'>
+                                        <Localize
+                                            i18n_default_text='I agree to move my MT5 account(s) and agree to Deriv BVI Ltd’s <0>terms and conditions</0>'
```

Comment:
Words such as `MT5` and `Deriv BVI Ltd` should be in a constant/config file.
@amina-deriv please help to coordinate for such thing and discuss it to all the team members to follow the same standard.